java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/Statement.java

```diff
@@ -0,0 +1,314 @@
+package com._4paradigm.openmldb.jdbc;
+import com._4paradigm.openmldb.SQLRouter;
+import com._4paradigm.openmldb.Status;
+
+import java.sql.*;
+
+public class Statement implements java.sql.Statement {
+    private com._4paradigm.openmldb.ResultSet resultSet;
+    private SQLRouter sqlRouter;
+    private boolean closed;
+
+    public Statement(SQLRouter router) {
+        this.sqlRouter = router;
+        this.closed = false;
+    }
+
+    @Override
+    public boolean execute(String sql) throws SQLException {
```

Comment:
Some sql like `use db` may not return `resultSet`. Will it return to false here even if executed successfully?

Reply:
- if execute failed, it will throw an exception.
- true if the first result is a ResultSet object; false if there are no results
https://docs.oracle.com/javase/7/docs/api/java/sql/Statement.html#execute(java.lang.String)

Reply:
Okay.